CMakeLists.txt

```diff
@@ -1915,3 +1915,9 @@ if(ENABLE_NO_EXPORT)
   message(STATUS "Building with empty ZMQ_EXPORT macro")
   add_definitions(-DZMQ_NO_EXPORT)
 endif()
+
+if (ENABLE_CURVE)
+  add_executable(curve_keygen tools/curve_keygen.cpp)
+  target_link_libraries(curve_keygen libzmq)
+  #target_include_directories(curve_keygen PUBLIC "${CMAKE_CURRENT_LIST_DIR}/include")
```

Comment:
leftover commented out line?

Reply:
also not a cpack person, no idea if it should be expected in a cpack.....